src/Mutator/Boolean/LogicalOr.php

```diff
@@ -77,6 +79,76 @@ public function mutate(Node $node): iterable
 
     public function canMutate(Node $node): bool
     {
-        return $node instanceof Node\Expr\BinaryOp\BooleanOr;
+        if (!$node instanceof Node\Expr\BinaryOp\BooleanOr) {
+            return false;
+        }
+
+        $equalOp = [
+            Node\Expr\BinaryOp\Identical::class,
+            Node\Expr\BinaryOp\Equal::class,
+        ];
+
+        if (
+            in_array(get_class($node->left), $equalOp, true) === true
```

Comment:
As you wish. I always add a strict comparison because sometimes we want it to be equal to false and the `!` is easy to miss / remove. I'll adapt to your coding style.